File: src/CrowdfindingApp.Common/DataTransfers/Projects/DraftProjectInfo.cs
using System;

namespace CrowdfindingApp.Common.DataTransfers.Project
{
    public class DraftProjectInfo
    {
        public string Id { get; set; }
        public string CategoryId { get; set; }
        public string Title { get; set; }
        public string ShortDescription { get; set; }
        public string FullDescription { get; set; }
        public string Location { get; set; }
        public string VideoUrl { get; set; }
        public string Image { get; set; }
        public DateTime? StartDateTime { get; set; }
        public int? Duration { get; set; }
        public decimal? Budget { get; set; }
        public string AuthorSurname { get; set; }
        public string AuthorName { get; set; }
        public DateTime? AuthorDateOfBirth { get; set; }
        public string AuthorMiddleName { get; set; }
        public string AuthorPersonalNumber { get; set; }
        public string WhomGivenDocument { get; set; }
        public DateTime? WhenGivenDocument { get; set; }
        public string AuthorAddress { get; set; }
        public string AuthorPhone { get; set; }
    }
}

File: src/CrowdfindingApp.Client/src/app/models/replies/projects/DraftProjectInfo.ts
import { RewardInfo } from '../rewards/RewardInfo';
import { QuestionInfo } from './QuestionInfo';

export class DraftProjectInfo {
  public id: string;
  public categoryId: string;
  public title: string;
  public shortDescription: string;
  public fullDescription: string;
  public location: string;
  public videoUrl: string;
  public image: string;
  public startDateTime: Date;
  public duration: number;
  public budget: number;
  public authorSurname: string;
  public authorName: string;
  public authorDateOfBirth: string;
  public authorMiddleName: string;
  public authorPersonalNumber: string;
  public whomGivenDocument: string;
  public whenGivenDocument: Date;
  public authorAddress: string;
  public authorPhone: string;
  public rewards: RewardInfo[];
  public questions: QuestionInfo[];
}
